packages/react-native/sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |spec|
   spec.license     = package['license']
   spec.author      = "Facebook"
   spec.source      = source
-  spec.platforms   = { :osx => "10.13", :ios => min_ios_version_supported }
+  spec.platforms   = { :osx => "10.13", :ios => "13.4" }
```

Comment:
any specific reason why did variable had to be reverted to a fixed value?

Reply:
yes, the podspec is used in by the build scripts in bash and the `min_ios_version_supported` function is not available there because it is defined in the `react_native_pods.rb`.

I think we can export this to a separate ruby file to handle dependencies...